test/compiler/codegen.jl

```diff
@@ -451,3 +451,8 @@ end
     a = [1]; b = [2]
     @test ifelsetuple(5, 3, a, b) == ([1], [1], [1], [2], [2])
 end
+
+@testset "#36422" begin
+    str_36422 = "using InteractiveUtils; code_llvm(Base.ht_keyindex, (Dict{NTuple{65,Int64},Nothing}, NTuple{65,Int64}))"
+    @test success(`$(Base.julia_cmd()) --startup-file=no -e $str_36422`)
```

Comment:
Why is this in a new process? So it doesn't segfault the worker if it fails?

Reply:
We do have many tests that will just segfault the worker if they "fail", so I think it would be fine without the external process, but I don't care much either way.

Reply:
Yes, that's why. I saw this method used above, so I thought it would be desired. If I should change it I certainly can.